image_explorer/public/css/image_explorer.css

```diff
@@ -151,3 +151,7 @@ div.image-explorer-hotspot .image-explorer-hotspot-reveal .image-explorer-hotspo
     display: inline-block;
     width: 90%;
 }
+
+.xblock-render{
+    overflow: unset !important;
```

Comment:
Need to use `overflow: visible !important` for this to work in IE11.  `unset` doesn't seem to be supported.